packages/rum-core/src/common/config-service.js

```diff
@@ -66,52 +72,53 @@ class Config {
   constructor() {
     this.config = {}
     this.defaults = {
-      serviceName: '',
-      serviceVersion: '',
-      environment: '',
+      // String Opts
```

Comment:
I did the other way around because if we decided to remove config service class and use a POJO for `config.js`, It would be ideally easier to have normalize for each options like `String`, `Boolean` and so on.. which makes our refactoring easier in future.